indico/modules/events/registration/util.py

```diff
@@ -158,7 +163,8 @@ def validate_email(self, field):
     for form_item in regform.active_fields:
         if not management and form_item.parent.is_manager_only:
             continue
-
+        if not all(values_from_signal(signal.send(regform, section=form_item.parent, registration=registration))):
```

Comment:
You use it in 3 places and it gets called for each form item. What about moving it to a util like this?

```python
@memoize_request
def _is_section_visible(regform, section, registration=None):
    return all(values_from_signal(signal.send(regform, section=section, registration=registration)))
```

That way you don't spam the signal that much in larger registration forms.